Packs/Salesforce/Integrations/Salesforce_IAM/README.md

```diff
@@ -0,0 +1,260 @@
+> <i>Note:</i> This integration should be used along with our IAM premium pack. For further details, visit our IAM pack documentation.
+
+Integrate with Salesforce's services to perform Identity Lifecycle Management operations.
+For more information, please refer to the [Identity Lifecycle Management article](https://xsoar.pan.dev/docs/reference/articles/identity-lifecycle-management).
+
+## Configure Salesforce IAM on Cortex XSOAR
```

Comment:
as part of the configuration we should mention the required fields and point them to mappers and the automations of profileid

Reply:
Added a paragraph about the mandatory fields to both, read me and description.